docs/docs/examples/llm/cleanlab.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    llm = CleanlabTLM(
</pre><p>Can you compact these three lines into a single one: <code>llm = CleanlabTLM(api_key="your_api_key")</code>? The linter is complaining  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/13994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>